apps/server/src/stores/EventStore.ts

```diff
@@ -32,10 +32,11 @@ export const eventStore = {
     //if there is already and update pending we don't need to schedule another one
     if (!isUpdatePending) {
```

Comment:
I wonder what would be the design differences in letting the consumer control this

ie: event store could expose a batch function, which returns a closure with a send function.
in that way the consumer would do 

```ts
const batch = eventStore.createBatch()

batch.add(timer, {});
batch.add(clock, {});
...
batch.send()
```

The immediate advantage is that we are sure that the patches are under our control and we know what is in them, the current solution could mix patches between sources

I also wonder if the logic would be simpler and not rely on scheduling magic

What do you think?